AdminApi/http/Identities/Deletion Processes/Cancel Deletion Process.bru

```diff
@@ -0,0 +1,16 @@
+meta {
+  name: Cancel Deletion Process
+  type: http
+  seq: 4
+}
+
+put {
+  url: {{BaseUrl}}/Identities/{{IdentityAddress}}/DeletionProcesses/{{DeletionProcessId}}/Cancel
+  body: none
+  auth: none
+}
+
+vars:pre-request {
+  IdentityAddress: id1CsU6VwBMgBA4TLc61nj895ZD1mPjbddNW
+  DeletionProcessId: IDPHEZQYas1NdHto2GU7
```

Comment:
AFAIK, not in the current version

Reply:
Can you do a quick research? Just to be sure

Reply:
I don't think the env files are a good solution. They are probably worse than the current solution. Because you always have to open the env file in parallel.
Then we better commit/undo the changed bruno files

Reply:
ok